### ISSUE TITLE
Accept multiple PATH in the CLI

### ISSUE THREAD

**Issue by AliSoftware** [AliSoftware/SwiftGen#33]
We should accept an infinite number of input PATH from the CLI (instead of only one), and iterate over all of them to feed our `EnumBuilder` before generating a big output containing them all

**Comment by AliSoftware** [comment]
@kylef How could we achieve this (parsing an arbitrary number of args at the end of the command) using `Commander`?

**Comment by kylef** [comment]
@AliSoftware Just to clarify, is that with an argument or a flag?

```
tool --path a b c
```

```
tool a b c
```

**Comment by AliSoftware** [comment]
Well both actually, I expect `swiftgen` to accept optional flags like, say, `--verbose`, accept optional arguments like `--output path`, and once all are consumed, I want every `Argument` left to be parsed.

```
$ swiftgen PATH1 PATH2 PATH3
$ swiftgen --verbose PATH1 PATH2 PATH3
$ swiftgen --output OUTPATH PATH1 PATH2 PATH3
```

I expect to be able to write something probably like this:
 - `command(…) { verboseFlag, output, pathsArray in … }`
or maybe
 - `command(…) { verboseFlag, output, parser in … let strings: [String] = parser.parseRemaining() }`

Is this kind of stuff possible?

**Comment by AliSoftware** [comment]
Implemented by @djbe in #213
